src/pipx/commands/install.py

```diff
@@ -30,6 +30,12 @@ def install(
         package_name = package_name_from_spec(
             package_spec, python, pip_args=pip_args, verbose=verbose
         )
+
+    override_shared = False
+
+    if package_name in ["pip", "wheel"]:
+        override_shared = True
```

Comment:
`pipx install pip` does not really make practical sense IMO so I’d recommend only handle `wheel` until someone proposes a use case.

Reply:
Thanks for the review. Updated.